grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -540,7 +540,14 @@ public static HttpHeaders statusToTrailers(
         GrpcTrailersUtil.addStatusMessageToTrailers(
                 trailersBuilder, status.getCode().value(), status.getDescription());
 
-        MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        try {
+            MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        } catch (Exception e) {
+            // Catching exception is necessary, because server implementer could have set corrupted metadata.
+            return trailersBuilder
+                    .set(GrpcHeaderNames.GRPC_STATUS, "13")
```

Comment:
Can we extract the `13` to a meaningful field variable so that others don't have to look up the code value to know what this means?